src/Compatibility/Microsoft.DotNet.ApiCompatibility/Microsoft.DotNet.ApiCompatibility.csproj

```diff
@@ -3,9 +3,8 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <IsPackable>true</IsPackable>
```

Comment:
Actually should we set IsPackable to false, or do you have a scenario for consuming the library today?

Reply:
I was going to remove it, but we might end up the need to ship it if we want to build more tooling on top of it that doesn't leave on the SDK? (Analyzer?). I don't mind stop producing a package on the meantime though.